appveyor.yml

```diff
@@ -26,6 +26,12 @@ environment:
     - TOXENV: py37-defaultreactor, win-py37-qt5reactor, py37-asyncioreactor
       PYTHON: "C:\\Python37-x64"
 
+    - TOXENV: py38-defaultreactor, win-py38-qt5reactor, py38-asyncioreactor
+      PYTHON: "C:\\Python37"
```

Comment:
Shouldn't this be Python38?

Reply:
*sigh*  bonus, I forgot I even had this open.

Reply:
Oh right...  I started this, noticed the failure and said "wait just a minute here...  Twisted doesn't even 'support' 3.8!"  So I went and thought hey, that's an easy PR...  until I realized you can't PR because of Azure web config.  So I walked away and gave up.